docs/user-guide/data.md

```diff
@@ -220,6 +220,98 @@ archive and twice in the path on the new /home file system.
 
     The capacity of the home file system is much larger than the work file system so you should move data to home rather than work.
 
+
+
+## Sharing data with other Cirrus users
+
+How you share data with other Cirrus users depends on whether or not
+they belong to the same project as you. Each project has two shared
+folders that can be used for sharing data.
+
+### Sharing data with Cirrus users in your project
+
+Each project has an *inner* shared folder.
+
+    /work/[project code]/[project code]/shared
+
+This folder has read/write permissions for all project members. You can
+place any data you wish to share with other project members in this
+directory. For example, if your project code is x01 the inner shared
+folder would be located at `/work/x01/x01/shared`.
+
+### Sharing data with  Cirrus users within the same project group
+
+Some projects have [subprojects](#subprojects) (also often referred to as a 'project groups' or sub-budgets)   e.g. project e123 might have a project group e123-fred  for a sub-group of researchers working with Fred.
+
+Often project groups do not have a disk quota set, but if the project PI [does set up a group disk quota](https://epcced.github.io/safe-docs/safe-for-managers/#how-can-i-create-a-quota-for-a-project-group-or-move-space-between-quotas) e.g. for /work then additional directories are created:
+
+	/work/e123/e123-fred
+	/work/e123/e123-fred/shared
+	/work/e123/e123-fred/<user> (for every user in the group)
+
+and all members of the ```/work/e123/e123-fred``` group will be able to use the ```/work/e123/e123-fred/shared``` directory to share their files.
+
+!!! Note
+    If files are copied from their usual directories they will keep the original ownership. To grant ownership to the group:
```

Comment:
I'd probably avoid phrases such as 'to grant ownership to the group' (the owner is always the user, and users can't use `chown` to change the owner). One could say e.g., 'to change the group the file belongs to...'.